### DISCUSSION TITLE
How to set default config?

### DISCUSSION THREAD

**Discussion by yutontaku** [fastfetch-cli/fastfetch#1358] [Q&A]
I want to set a default config but don't know how
Writing fastfetch -c /path/to/config is long

**Comment by Zeroes1** [comment]
fastfetch.exe --gen-config-force
The generated default config file has written in `C:/ProgramData/fastfetch/config.jsonc`

to load
fastfetch.exe -c path2config

you can make Bat file for fast run

for Linux/... same actions

**Comment by LinuxOnTheDesktop** [comment]
Ah. So the program cannot _have_ a default configuration, strictly speaking (unless one counts the as it were inbuilt - the file-less - configuration)
